tests/seed/accounts/test_licensee.py

```
@@ -65,63 +25,45 @@ def test_build_licensee_data(context: Context, monkeypatch):
-    mock_get_licensee = mocker.patch(
-        "seed.accounts.licensee.get_licensee", new_callable=mocker.AsyncMock
-    )
-    mocker.patch(
-        "seed.accounts.licensee.build_licensee_data", return_value=build_licensee_data(context)
-    )
-    mock_get_licensee.return_value = None
-    fs.create_file("/mpt_api_client/seed/data/logo.png", contents=b"fake_icon_bytes")
-    result = await init_licensee(context=context, mpt_client=client_client)
-    assert result == licensee
-    licensees_service.create.assert_called_once()
+async def test_create_licensee(mocker, client_client, context_with_data):  # noqa: WPS218
```

Ruff (0.14.8)

Unused `noqa` directive (unknown: `WPS218`)

Remove unused `noqa` directive

(RUF100)